fast_llm/layers/language_model/config.py

```diff
@@ -118,6 +118,16 @@ class LanguageModelBaseConfig(LanguageModelArchitectureConfig, BaseModelConfig):
         hint=FieldHint.feature,
         valid=check_field(Assert.geq, 0),
     )
+    init_method_max_embed: float|None = Field(
```

Comment:
The spacing is wrong, it should be `float | None`. Did you set up pre-commit?

Reply:
ah, thanks! Just setup pre-commit now